src/llvm-late-gc-lowering.cpp

```diff
@@ -2594,7 +2594,7 @@ void LateLowerGCFrame::PlaceGCFrameStore(State &S, unsigned R, unsigned MinColor
     auto slotAddress = CallInst::Create(
         getOrDeclare(jl_intrinsics::getGCFrameSlot),
         {GCFrame, ConstantInt::get(Type::getInt32Ty(InsertBefore->getContext()), Colors[R] + MinColorRoot)},
-        "", InsertBefore);
+        "gc_slot_addr", InsertBefore);
```

Comment:
Numbering the name of the gc_slot_addr can probably help with debugging/optimizing late-gc-lowering too.